### PR TITLE
Updated Tracy

### PR DESCRIPTION
**Pull request by xCryptic**
• Added Gift Weights
• Added Upgrading
• Added Greetings
• Added missing keywords
• Updated `@talk`
• Updated Intro for #241
• Updated about_skill keyword
• Fixed ModifyRelation values
• Other small fixes

### CODE REVIEW

File: system/scripts/npcs/dugald/tracy.cs [hunk 1]
```diff
@@ -197,14 +289,15 @@ public override void Load()
...
+
 				switch (await Select())
 				{
 					case "@yes":
 						Msg("Awesome! Repeat after me!<br/>[Tough Guy Tracy!]<br/>Say it 100 times and I'll give you a piece of bread.", Button("Tough Guy Tracy!", "@reply1"), Button("Crazy Dude", "@reply2"));
+
 						switch (await Select())
 						{
 							case "reply1":
-								// NOTE: He asks you to say it 100 times, but I received this message after 1 time
```
Comment (xCryptic) [inline system/scripts/npcs/dugald/tracy.cs]: This was intentional. No one would click through it 100 times anyway.

Comment (exectails) [inline system/scripts/npcs/dugald/tracy.cs]: Where is the `Give(bread)` though?^^

Comment (xCryptic) [inline system/scripts/npcs/dugald/tracy.cs]: I didn't get any bread on NA. Do you think they would just give away free bread? That would be spammable.

Comment (exectails) [inline system/scripts/npcs/dugald/tracy.cs]: I wasn't serious^^ I don't think you're supposed to get bread, based on what he says.

Comment (xCryptic) [inline system/scripts/npcs/dugald/tracy.cs]: Oh... welp.

**Pull request merged**